researchstudio-sat/webofneeds
# Feat  rdf in chat

**Test:**

1. in an open connection, click "show rdf"
2. send a message with arbitrary rdf using the send-rdf-textfield. e.g.:
```trig
@prefix c: <http://example.org/cartoons#>.
c:Tom a c:Cat.
<this:messageuri> a c:Mouse;
        c:smarterThan c:Tom.
c:Jerry a c:Mouse;
        c:smarterThan c:Tom.
```
3. on the receiving side, also click "show rdf", you should see a trig represenation of that.
4. click the message, you should see the messages `@context` in trig format (i.e. a bunch of statements starting with `@prefix`)

**NOTE:** plz don't delete the branch yet. the `@prefix`-thing still needs a visual indicator and i still need to clean up the textfield. This PRs intention is to get the results so far into upstream / avoid divergance.
